### ISSUE TITLE
monocode is getting stripped when its no enabled 

### ISSUE THREAD

**Issue by aicynide** [Charibdys/Private-Parlor#112]
* **What is the current behavior?**
I've not enabled monocode stripping in config but its gettin stripped in /g/ bot

* **If the current behavior is a bug, please provide the steps to reproduce it**

* **What is the expected behavior?**

* **What is the motivation / use case for changing the behavior?**

* **Please tell us about your environment:**
  
  - Version: 0.6.X
  - Operating System: Puppy Linux FossaPup64 9.5
  - Compiled from source: Yes, using Crystal 1.9.0/No, using v0.6 binaries from GitHub

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix)

**Comment by Charibdys** [comment]
I'm going to need more information. The form is not filled out completely.

If you are actually using version 0.6 nightly of Private Parlor then I cannot help you as message entity stripping was reworked in a later version.

In version 1.1, entities should not be stripped unless explicitly defined in the config. Please show the values you have changed in the config.yaml file and describe the steps needed to reproduce this bug.

**Comment by aicynide** [comment]
Yes I'm using v1.1 and monocode is getting stripped when its not enabled
In case you aren't aware about monocode its `monocode like this`
The text between two `

**Comment by aicynide** [comment]
Yes this this called monocode https://github.com/Charibdys/Private-Parlor/blob/master/config.yaml.copy#L217

**Comment by Charibdys** [comment]
Are you using the client's text formatting options?
On desktop this is done by selecting the text, right clicking, selecting formatting, then choosing "Monospace"
Or on mobile clients this is done by selecting the text, holding, then choosing "Monospace"

If you are using Markdown or HTML (<code>text here</code>) then this is not possible in version 1.1.
